### DISCUSSION TITLE
IOS客户端显示网络有问题，但是windows就没问题，日志也没有问题

### DISCUSSION THREAD

**Discussion by golowper** [1715173329/UnblockNeteaseMusic#681] [Q&A] [answered]
大佬棒棒
http和https都会出现相同的问题
![bad12736ba44ac699e3e6f667493195](https://user-images.githubusercontent.com/77011000/167858615-627673a3-010d-4970-a54a-abf0e6570256.jpg)

这是我的日志
![image](https://user-images.githubusercontent.com/77011000/167858563-e50035c1-b288-41c7-b236-58650f8b9ec3.png)

**Comment by 1715173329** [comment, marked as answer]
1. 信任证书
2. 设置 endpoint，具体方法可以搜索 issue

**Reply by golowper** [reply]
大佬我证书装好了，我太菜了没找到ios使用endpoint的issue不太会用，可以细说一下如何使用endpoint吗🥹

**Reply by golowper** [reply]
不用了，弄好了 谢谢大佬 原来在readme里
